### PR TITLE
Filter accordion items to only display categories that contain services

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**  
Only display categories that have services associated with them:
Before:
image
After:
image

**Why**  
To remove irrelevant headings for empty categories

### CODE REVIEW

File: components/Feature/Services/index.js [hunk 1]
```diff
@@ -1,48 +1,50 @@
...
-  AccordionItem
-} from 'components/Form';
+import { Accordion, AccordionItem } from 'components/Form';
 
 const Services = ({ resources, taxonomies }) => {
   const [expandedGroups, setExpandedGroups] = useState({});
 
+  taxonomies.forEach(
+    taxonomy =>
+      (taxonomy.resources = resources
```
Comment (antonyoneill) [inline components/Feature/Services/index.js:10]: I don't think you should mutate the object that you've received through the props here.

Is there a clearer way you can write this?

Comment (kosiakkatrina) [reply components/Feature/Services/index.js:10]: Yeah, I guess I could copy it Into another object 🤷

Comment (antonyoneill) [reply components/Feature/Services/index.js:10]: For example, you could have `taxonomiesToRender` which does something like

```
const taxonomiesToRender = taxonomies.filter(
  taxonomy => resources.some(
    resource => resource.categoryName === taxonomy.name
  )
)
```

Comment (kosiakkatrina) [reply components/Feature/Services/index.js:10]: Oooh, I like it a lot. I forget that world outside of map exists 😂 I'll add that now

Comment (kosiakkatrina) [reply components/Feature/Services/index.js:10]: Hmm, now to think of it, this forces to have ``` resource => resource.categoryName === taxonomy.name``` check twice. Once here and also once as we render the resources. I'm tempted to put it into one object before the render, e.g. like this:
```
  const taxonomiesToRender = taxonomies.filter(taxonomy =>
    resources.some(resource => resource.categoryName === taxonomy.name)
  );

  taxonomiesToRender.forEach(
    taxonomy =>
      (taxonomy.resources = resources
        .map(resource => {
          if (taxonomy.name == resource.categoryName) return resource;
        })
        .filter(x => x))
  );
  ```
But that almost seems like the first part of it is irrelevant. Any ideas?

**Review by ben-d758 — APPROVED** [review]

**Pull request merged**